main/realmain.cc

```diff
@@ -162,6 +162,15 @@ core::Loc findTyped(unique_ptr<core::GlobalState> &gs, core::FileRef file) {
 }
 
 #ifndef SORBET_REALMAIN_MIN
+class LSPStdout final : public lsp::LSPOutput {
```

Comment:
WDYT about moving this into `LSPOutput` header as one of default implementations?